public/app/features/api-keys/ApiKeysPage.tsx

```diff
@@ -89,15 +100,28 @@ export class ApiKeysPageUnconnected extends PureComponent<Props, State> {
 
   onMigrateApiKeys = async () => {
     try {
-      this.onMigrateAll();
```

Comment:
Should we remove the method as well?

Reply:
yea that's true it's not being used anymore